.gitignore

```diff
@@ -0,0 +1,11 @@
+**/target
+auth.rs
```

Comment:
curious why gitignore the `.rs` files?

Reply:
imo we should dump the build output to a different folder or just commit these files